### PR TITLE
add count support for options + onCellClick callback

### PR DESCRIPTION
**Pull request by gregnb**
This should solve #75 and also #73 when merged 

@rst66 @adetwiler @jstgermain

### CODE REVIEW

File: src/MUIDataTableBody.js [hunk 1]
```diff
@@ -115,6 +122,8 @@ class MUIDataTableBody extends React.Component {
             <MUIDataTableBodyCell
               colSpan={options.selectableRows ? columns.length + 1 : columns.length}
               options={options}>
```
Comment (jstgermain) [inline src/MUIDataTableBody.js]: this closing HTML bracket should be after the `rowIndex`

Comment (gregnb) [reply src/MUIDataTableBody.js]: Nice catch. Thank you!

**Comment by gregnb** [comment]
@jstgermain I haven't given that much thought but if you want to spearhead that initiative and provide and example I would love to include it into examples/ 

If you're stuck maybe at least create a test repo with some mock data and we can work from there? See if we can come up with a viable solution.

I will be merging this PR later tonight and doing a release then

**Comment by jstgermain** [comment]
@gregnb Awesome! I am a dev for a small project, and @adetwiler is actually the CTO for the project. We would love to work on this with you. If you could contact me via text message on my business line, (480) 757-9953, maybe we can setup a hangout to show you what we are dealing with and come up with a solution together. We are interested in assisting in the development of this project. It is amazing and would love to think of other edge cases to crush.

**Comment by gregnb** [comment]
@jstgermain Sure man, l'll touch base with you sometime tomorrow as I have a pretty full day with meetings. I'll try to assist as much as I can

**Comment by jstgermain** [comment]
@gregnb awesome. We look forward to it.

**Pull request merged**